hmy/downloader/shortrange.go

```diff
@@ -56,7 +58,7 @@ func (d *Downloader) doShortRangeSync() (int, error) {
 	}
 	n, err := d.ih.verifyAndInsertBlocks(blocks)
 	if err != nil {
-		if !errors.As(err, &sigVerifyError{}) {
+		if !errors.As(err, &emptySigVerifyError) {
```

Comment:
is emptySigVerifyError an pointer?

Reply:
It is, and it is OK to use a pointer to pointer here. The second argument need to be a pointer to error interface. Please check out the [example here](https://golang.org/pkg/errors/#example_As).